src/integrationTest/resources/tests/audio/MediaControllerPostAdminMediasSearchIntTest/emptyCourthouses.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
I think we need to pick a way of managing integration tests and the associated json? There are different ways at the moment:-

1) Many of the existing search tests serialize the request and deserialized the response. 
2) Some of the search tests use json path expressions to assert the values. 
3) Some use json files

if files are the way to go we should decide to add files for all of those searches for consistency.

Reply:
Dont worry about this comment. I will raise this on the best practices channel.

Reply:
For the difference search scenarios, would an alternative be to simply assert that the response contains the expected media ids?

A single test could cover that all other attributes are present and correct (but just for one scenario, with one or two search results).

I think this could help maintainability here - otherwise if we were to add a new field to the response we'll have quite a lot of JSON to update here.

Reply:
updated